submissions/adeyemiJuwon/phpDatabase/customers.php

```diff
@@ -0,0 +1,45 @@
+<?php
+try{
+$pdo = new PDO("mysql:host=localhost;dbname=store", 'root', 'mysql');
+     $pdo->setAttribute(Pdo::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    
+ }catch(PDOException $e){
+     die("ERROR:Could not connect." . $e->getMessage());
+ }
+  setcookie("Id",  $row['id'], time()+3600, "/", "", false);
```

Comment:
Where is `$row` coming from?

Reply:
Coming from the table from the database. @adaugochi

Reply:
@adaugochi. I have reformatted my code. help me to review it again. Thanks